src/GitHub.App/ViewModels/PullRequestCreationViewModel.cs

```diff
@@ -53,7 +55,9 @@ public class PullRequestCreationViewModel : BaseViewModel, IPullRequestCreationV
                 {
                     // what do we do if there's no master?
                     TargetBranch = x.FirstOrDefault(b => b.Name == "master");
-                    SourceBranch = x.FirstOrDefault(b => b.Name == repo.Head.FriendlyName);
+
+                    // 'repo.Head' will be null in unit tests.
+                    SourceBranch = x.FirstOrDefault(b => b.Name == repo.Head?.FriendlyName);
```

Comment:
For clarification: `VisualStudio.Services.DefaultExportProvider` called in https://github.com/github/VisualStudio/blob/master/src/GitHub.Exports/Services/GitService.cs#L106 goes to https://github.com/github/VisualStudio/blob/master/src/GitHub.Exports/Services/Services.cs#L45 which grabs the `ComponentModel` property in https://github.com/github/VisualStudio/blob/master/src/GitHub.Exports/Services/Services.cs#L44 which is looked up in `GetGlobalService` here: https://github.com/github/VisualStudio/blob/master/src/GitHub.Exports/Services/Services.cs#L32-L42.

When running unit tests, `GetGlobalService` uses the mocked `IServiceProvider` created in https://github.com/github/VisualStudio/blob/master/src/UnitTests/Substitutes.cs#L103-L111, which includes mocks of all our services plus the `ComponentModel` object and other infrastructure stuff. Anything that queries a service provider or goes through `VisualStudio.Services` properties ends up in these mocked objects.